File: SEP3_UserSystem/src/main/java/com/example/SEP3_UserSystem/mediator/information/ErrorPackage.java
package com.example.SEP3_UserSystem.mediator.information;

public class ErrorPackage extends InformationPackage{
    private boolean Error;

    public ErrorPackage(String errorMessage,boolean error)
    {
        super(InformationType.ERROR,errorMessage);
        Error = error;
    }

    public boolean isError() {
        return Error;
    }
}

File: SEP3_ChatSystem/Mediator/Information/ErrorPackage.cs
namespace SEP3_ChatSystem.Mediator.Information
{
    public class ErrorPackage : InformationPackage
    {
        public bool Error { get; set; }

        public ErrorPackage(string errorMessage,bool error) : base(InformationType.ERROR,errorMessage)
        {
            Error = error;
        }

        public ErrorPackage() {}

        public bool IsError() {
            return Error;
        }
    }
}